timemachine/fe/single_topology.py

```diff
@@ -568,12 +642,59 @@ def assert_improper_idxs_are_canonical(all_idxs):
     )
 
 
+def find_chirally_consistent_dummy_groups(
+    mol_a, mol_b, core: NDArray
+) -> Optional[Tuple[Dict[int, FrozenSet[int]], Dict[int, FrozenSet[int]]]]:
+    """Returns a pair of dummy group assignments for the A -> B and B -> A transformations such that the implied hybrid
+    mol is chirally consistent, or None if no such pair exists.
+
+    Refer to :py:func:`assert_chiral_consistency_and_validity` for definition of "chiral consistency".
+
+    Refer to :py:func:`timemachine.fe.dummy.generate_dummy_group_assignments` and notes below for more
+    information on dummy group assignment.
+
+    For the A -> B transformation, dummy group indices refer to atoms in mol_b.
+    For the B -> A transformation, dummy group indices refer to atoms in mol_a.
+    """
+
+    bond_graph_a = convert_to_nx(mol_a)
+    bond_graph_b = convert_to_nx(mol_b)
+
+    ff = Forcefield.load_from_file("placeholder_ff.py")
+
+    return find_chirally_consistent_dummy_groups_impl(mol_a, mol_b, bond_graph_a, bond_graph_b, core, ff)
+
+
+def find_chirally_consistent_dummy_groups_impl(
+    mol_a, mol_b, bond_graph_a: nx.Graph, bond_graph_b: nx.Graph, core: NDArray, ff: Forcefield
```

Comment:
This was intentional, for efficiency in the case where this is used in the MCS leaf filter:

https://github.com/proteneer/timemachine/blob/755bae826ee9db0a0f716c4776ee2b75e1530838/timemachine/fe/atom_mapping.py#L411-L426

There we precompute the bond graph, avoiding the need to call `convert_to_nx` at every leaf. This may have been a premature optimization, though.